assets/pages/bundles-landing/reducers/__tests__/reducersTest.js

```diff
@@ -28,9 +28,9 @@ describe('reducer tests', () => {
 
```

Comment:
Should we add tests for the common reducer?

Reply:
270a81d